swap_meet/clothing.py

```diff
@@ -1,2 +1,8 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, age=0, condition=0):
+        super().__init__(age, condition, category="Clothing")
```

Comment:
I like how you removed `category` from the constructor's parameters. We don't want the caller to be able to set a different category. It's entirely possible the caller could change the category later after an instance is created,  but at least we're indicating that they shouldn't do that.